src/block/column/style.scss

```diff
@@ -1,7 +1,7 @@
 @import "common";
 
 .stk-block.stk-block-column .stk-column-wrapper {
-	width: 100%;
+	width: auto;
```

Comment:
I'm not that comfortable with this fix because this would affect all Stackable column blocks, even in the frontend.

Since the issue in https://github.com/gambitph/Stackable/issues/2408 is only in the editor, it might be better if the fix only affects the editor